<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="PG014UCR" name="com.backbase.CreateNewComputerTestCases.Add_New_Computer_Functionality_TestCases" tests="1" failures="1" timestamp="21 Oct 2018 12:10:46 GMT" time="0.000" errors="0">
  <testcase name="TC_01_02_03_Verify_New_Computer_Addition_With_Valid_Data" time="0.000" classname="com.backbase.CreateNewComputerTestCases.Add_New_Computer_Functionality_TestCases">
    <failure type="org.testng.TestNGException" message="
DataProvider should be static: public java.lang.Object[][] com.backbase.utilLibrary.DataProviderRepository.getNewCompData10()">
      <![CDATA[org.testng.TestNGException: 
DataProvider should be static: public java.lang.Object[][] com.backbase.utilLibrary.DataProviderRepository.getNewCompData10()
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:328)
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:262)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:414)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:37)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- TC_01_02_03_Verify_New_Computer_Addition_With_Valid_Data -->
</testsuite> <!-- com.backbase.CreateNewComputerTestCases.Add_New_Computer_Functionality_TestCases -->
